src/compiler/scanner.ts

```diff
@@ -2455,6 +2457,35 @@ export function createScanner(languageVersion: ScriptTarget,
         return scanJsxAttributeValue();
     }
 
+    /** TODO: might need to return WhitespaceTrivia if only whitespace was encountered? */
+    function scanBigJsDocToken(): JSDocSyntaxKind { // can be configurable to skip almost everything (except newline and backtick) if backticks is true
+        startPos = tokenPos = pos;
+        tokenFlags = TokenFlags.None;
+        if (pos >= end) {
+            return token = SyntaxKind.EndOfFileToken;
+        }
+
+        let ch = codePointAt(text, pos);
+        while (pos < end) {
+            if (ch !== CharacterCodes.lineFeed && ch !== CharacterCodes.at && ch !== CharacterCodes.backtick && ch !== CharacterCodes.openBrace) {

```

Comment:
carriage return

Reply:
ah, I misread the code in scanJsDocToken. I blame `// falls through`.